### PR TITLE
Fix dir structure

### PR DESCRIPTION
**Pull request by pbeltran82**
Please merge @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2085)

### CODE REVIEW

File: english/eating-walls/find-a-toilet/magazine.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the time you get to spend reading something enjoyable. Makes time in the toilet enjoyable.
```
Comment (casey-collab) [inline english/eating-walls/find-a-toilet/magazine.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/eating-walls/find-a-toilet/magazine.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pbeltran82** [comment]
thank you

**Comment by cbuckey-uda** [comment]
Closing this pul request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**